static/main.js

```diff
@@ -17,14 +18,27 @@ window.onload = () => {
     cookieBanner.classList.add("visible");
   }
 
-  hideButton.addEventListener("click", () => {
-    const date = new Date();
-    date.setFullYear(date.getFullYear() + 1);
+  if (cookieBanner) {
```

Comment:
what happens if a user has js disabled? IIRC gov websites have to work fine without it.

Reply:
Oh. None of the buttons will work 😅 I guess we need to refactor the whole thing..?!

Reply:
@CheshireSwift ?

Reply:
yeah I think so. It's probably fine is no js -> just disallow cookies and dont show the thing

Reply:
From Sam: The buttons need to be a form, probably one that does a POST to the server. There can be a hidden input on the form saying what the current page is that we can use to redirect back to the same place.

Reply:
Yeah, no need for any JS here that I can see. HTML form, server handling.